Pages/Index.razor

```diff
@@ -1,7 +1,17 @@
 ﻿@page "/"
 
-<h1>Hello, world!</h1>
+<h1>Ask your question</h1>
 
-Welcome to your new app.
 
-<SurveyPrompt Title="How is Blazor working for you?" />
+<!DOCTYPE html>
```

Comment:
```suggestion

```
`DOCTYPE` tag should be removed (sorry not sure if I included that in the last review or not)